PYME/DSView/dsviewer.py

```diff
@@ -45,6 +45,10 @@
 
 import logging
 logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+logging.getLogger('matplotlib.font_manager').setLevel(logging.ERROR) #clobber unhelpful matplotlib debug messages
+logging.getLogger('matplotlib.backends.backend_wx').setLevel(logging.ERROR)
 
```

Comment:
I did not add the PIL clobber statement - not sure if really applicable to dsviewer? Can add if suitable.

Reply:
I'm not sure, but if you're not seeing any PIL related errors then it's probably fine.